jibri/rootfs/etc/cont-init.d/11-get-https-cert

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/with-contenv bash
+
+while ! openssl s_client -showcerts -connect meet.jitsi:443 < /dev/null | openssl x509 -outform PEM > /usr/local/share/ca-certificates/meet.crt; do sleep 1; done
+update-ca-certificates
+# the screenshot is just to launch chrome and exit, to make it create ~/.pki
+su jibri -s /bin/sh -c '/usr/bin/google-chrome --headless --screenshot=/tmp/s.png https://meet.jitsi/ && certutil -d sql:/home/jibri/.pki/nssdb -A -t TC -n meet -i /usr/local/share/ca-certificates/meet.crt'
```

Comment:
why is the screenshot to create the `~/.pki` required? I could not find any trace of this file being required on the net

Reply:
I haven't checked whether Chrome creates the database as empty. If so, your version would be simpler and faster.